src/lib/Search/PagerContentToDataMapper.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Search;
+
+use Pagerfanta\Pagerfanta;
+
+class PagerContentToDataMapper extends PagerContentToDataMapperAbstract
+{
+    /**
+     * @param Pagerfanta $pager
+     *
+     * @return array
+     */
+    public function map(Pagerfanta $pager): array
+    {
+        $data = [];
+        $contentTypeIds = [];
+
+        foreach ($pager as $content) {
+            /** @var \eZ\Publish\API\Repository\Values\Content\Content $content */
+            $contentInfo = $content->getVersionInfo()->getContentInfo();
```

Comment:
` $content->getVersionInfo()->getContentInfo()` could be simplified to: `$content->contentInfo`